src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -187,7 +231,11 @@ private String getVirtualTableName(final VirtualSchema virtualSchema, final Tabl
     }
 
     protected ResultSet query(final String sql) throws SQLException {
-        return connection.createStatement().executeQuery(sql);
+        try {
+            return connection.createStatement().executeQuery(sql);
+        } catch (final SQLException exception) {
+            throw new SQLException("Error executing '" + sql + "': " + exception.getMessage(), exception);
+        }
```

Comment:
Catching the exception here allows us to add the actual sql statement to the error message